chia/wallet/wallet_user_store.py

```diff
@@ -33,11 +33,17 @@ async def create(cls, db_wrapper: DBWrapper):
             )
         )
 
-        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS name on users_wallets(name)")
+        # this index is not used by any queries, don't create it for new
+        # installs, and remove it from existing installs in the future
+        # await self.db.execute("DROP INDEX IF EXISTS name on users_wallet(name)")
 
-        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS type on users_wallets(wallet_type)")
+        # this index is not used by any queries, don't create it for new
+        # installs, and remove it from existing installs in the future
+        # await self.db.execute("DROP INDEX IF EXISTS type on users_wallet(wallet_type)")
 
-        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS data on users_wallets(data)")
+        # this index is not used by any queries, don't create it for new
+        # installs, and remove it from existing installs in the future
+        # await self.db.execute("DROP INDEX IF EXISTS data on users_wallet(data)")
```

Comment:
it it's not actually used anywhere, I don't think there's any backwards-compatibility reason to keep it. We should probably just drop these indices here

Reply:
will delete these

Reply:
I'll remove them.